### PR TITLE
Simplify dark header template and pattern.

### PR DESCRIPTION
**Pull request by kjellr**
This PR simplifies the block setup for the Dark header template and pattern (used on the  homepage). 

- It relies on the recently-merged https://github.com/WordPress/gutenberg/pull/35589 to make the site tagline wide-width. Previously, the tagline was wrapped in a columns block that added a bunch of extra unnecessary margins. 
- Since that weird extra margin is gone, this PR also re-adjusts some of the paddings/margin to make things look a little bit more even. 

When testing, you'll need to be running the latest Gutenberg `trunk`. (If it works, I think it's fine to merge this in now, even though that change hasn't made it to the plugin yet). 

Before|After
---|---
Screen Shot 2021-10-13 at 09 01 33|Screen Shot 2021-10-13 at 09 01 06

### CODE REVIEW

File: inc/patterns/header-large-dark.php [hunk 1]
```diff
@@ -6,24 +6,19 @@
...
 					<div class="wp-block-group alignwide" style="padding-top:max(1.25rem, 5vw);padding-bottom:8rem"><!-- wp:group {"layout":{"type":"flex"}} -->
-					<div class="wp-block-group">
-					<!-- wp:site-logo {"width":64} /-->
+					<div class="wp-block-group"><!-- wp:site-logo {"width":64} /-->
 
 					<!-- wp:site-title {"style":{"typography":{"fontStyle":"italic","fontWeight":"400"}}} /--></div>
 					<!-- /wp:group -->
 
-					<!-- wp:navigation {"itemsJustification":"right","isResponsive":true} -->
+					<!-- wp:navigation {"itemsJustification":"right"} -->
```
Comment (jffng) [inline inc/patterns/header-large-dark.php]: Did you mean to remove the `isResponsive` attribute?

**Review by jffng — APPROVED** [review]
Just one comment about responsive menu on the nav, otherwise this LGTM.

File: inc/patterns/header-large-dark.php (hunk 1 above)
Comment (kjellr) [reply inc/patterns/header-large-dark.php]: I did not! Weird. I'll put it back in.

Comment (kjellr) [inline inc/patterns/header-large-dark.php]:
```suggestion
					<!-- wp:navigation {"itemsJustification":"right","isResponsive":true} -->
```

**Pull request merged**